### PR TITLE
add setting for 'git bash' mode

### LINKED ISSUE #62: support posix shells in Windows
**Is your feature request related to a problem? Please describe.**
If a user has git-bash as the default terminal on VSCode, the `okteto up` command won't be able to start.

### PR DESCRIPTION
**Pull request by rberrelleza**
fixes #62

### CODE REVIEW

File: src/okteto.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import * as os from 'os';
 import * as download from 'download';
 import * as semver from 'semver';
 import {pascalCase} from 'change-case';
+import *  as paths from './paths';
```
Comment (rlamana) [inline src/okteto.ts]: nit: remove whitespace

```suggestion
import * as paths from './paths';
```

File: src/okteto.ts [hunk 2]
```diff
@@ -144,7 +144,14 @@ export function start(manifest: string, namespace: string, name: string, port: n
     }
   });
 
-  term.sendText(`${getBinary()} up -f ${manifest} --remote ${port}`, true);
+  
+  var binary = getBinary();
```
Comment (rlamana) [inline src/okteto.ts]: block-scoped `let` is recommended instead of `var`.

```suggestion
  let binary = getBinary();
```

File: src/paths.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+import * as path from 'path';
+
+export function toGitBash(p: string): string {
+  const split = p.split(path.win32.sep);
+  console.log(split);
+  const joined = path.posix.join(...split);
+  var regex = /^([A-Za-z0-9]:).*/;
```
Comment (rlamana) [inline src/paths.ts]:
```suggestion
  let regex = /^([A-Za-z0-9]:).*/;
```

File: src/test/suite/paths.test.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+'use strict';
```
Comment (rlamana) [inline src/test/suite/paths.test.ts]: any particular reason you want strict mode with Typescript? isn't it already pretty strict?

Comment (rberrelleza) [reply src/test/suite/paths.test.ts]: I found that it catches more things when compiling. Is there any harm in having it?

**Pull request merged**